src/routes/About.js

```diff
@@ -19,8 +20,7 @@ export const About = () => {
             <div className="grid pt-8 text-left border-t border-gray-200 md:gap-16 md:grid-cols-1">
               <div className="justify-center mb-4 xl:hidden">
                 <picture>
-                  <source srcSet="bar.jpg" type="image/jpeg" />
-                  <img className="rounded-lg" src="/bar.jpg" alt="banner" />
+                  <img className="rounded-lg" src="/team.png" alt="banner" />
```

Comment:
Nit: just an idea but we could make the banner image change on click for mobile so we have the same Easter egg

Reply:
We can do that another time, Don't have time now to change it. I'm just gonna merge it so we actually have a photo of us on the site